init.lua

```diff
@@ -59,30 +60,36 @@ local attribute = {
 local function is_player(player)
 	return (
 		player and
-		not player.is_fake_player and
-		player.get_attribute and  -- check for pipeworks fake player
 		player.is_player and
-		player:is_player()
+		player:is_player() and
```

Comment:
player.is_player and
player:is_player() ?

Reply:
I presume, because is_player might not be defined for some objects that end up here.

Reply:
Or use `minetest.is_player` which should replace this entire function.

Reply:
Good call, but that will report true for the pipeworks fake player. But it can replace most of this function.